cibyl/sources/elasticsearch/api.py

```diff
@@ -144,6 +144,27 @@ def get_last_build(self, **kwargs):
 
         return AttributeDictValue("jobs", attr_type=Job, value=job_object)
 
+    def get_builds_by_status(self: object, **kwargs: str):
```

Comment:
I would use the general get_builds method for this case too. The user might want to filter by build id and status at the same time, so calling two different functions might give the wrong results if not all filters are applied at once.

Reply:
Mmm. In this case the method I’ve done uses `get_builds`, it loops over  the builds and delete of the dictionary all the builds that doesn’t match the status. Idk if it’s better to add more logic into the `get_builds` method

Reply:
I'll do the change, I've answered this one in: https://github.com/rhos-infra/cibyl/pull/97#discussion_r833809978 and continue there. Thx Joan.